Building Windows binary on 3.1.3 fails

Related https://github.com/kubernetes/minikube/pull/16866

```
Error: ../../../go/pkg/mod/github.com/cheggaaa/pb/v3@v3.1.3/pool_win.go:39:11: undefined: cols
Error: ../../../go/pkg/mod/github.com/cheggaaa/pb/v3@v3.1.3/pool_win.go:40:14: undefined: strings
```

--- a/v3/pool_win.go
+++ b/v3/pool_win.go
@@ -6,6 +6,7 @@ package pb
 import (
 	"fmt"
 	"log"
+	"strings"
 
 	"github.com/cheggaaa/pb/v3/termutil"
 )
@@ -29,6 +30,10 @@ func (p *Pool) print(first bool) bool {
 		if err != nil {
 			log.Panic(err)
 		}
+	}
+	cols, err := termutil.TerminalWidth()
+	if err != nil {
+		cols = defaultBarWidth
 	}
 	isFinished := true
 	for _, bar := range p.bars {

